### ISSUE TITLE
Do not throw exception for suspended/deleted account

### ISSUE THREAD

**Issue by nemobis** [robertoszek/pleroma-bot#71]
Arguably not a bug but user error:
> User "SaferInternetEU" has been suspended

https://nitter.eu/SaferInternetEU

```
ℹ 2022-04-03 18:30:42,941 - pleroma_bot - INFO - It seems like pleroma-bot is running for the first time for this Twitter user: SaferInternetEU 
✖ 2022-04-03 18:30:43,993 - pleroma_bot - ERROR - Exception occurred (cli.py:539) 
Traceback (most recent call last):
  File "/home/federico/mw/pleroma-bot/res/lib64/python3.10/site-packages/pleroma_bot/cli.py", line 422, in main
    user = User(user_item, config, base_path)
  File "/home/federico/mw/pleroma-bot/res/lib64/python3.10/site-packages/pleroma_bot/cli.py", line 211, in __init__
    self._get_twitter_info()
  File "/home/federico/mw/pleroma-bot/res/lib64/python3.10/site-packages/pleroma_bot/_twitter.py", line 45, in _get_twitter_info
    user = json.loads(response.text)["data"]
KeyError: 'data'
```

I'd prefer to just log such errors and proceed. During setup I can remove the accounts that shouldn't have been there in the first place, but in the future if a single account gets renamed or suspended everything will fail.

**Comment by robertoszek** [comment]
I'm a little bit torn whether this should raise an exception and stop execution or not.
On one hand, it's something that requires your attention and you should be aware it failed and why. But on the other hand, it could block you from continuing to use the bot if for whatever reason one of the users on your config fails.

So I see the appeal,
I think a good compromise it's to continue to execute and process the next users on the config but exit the app with an error exit code (and logging the exceptions and errors, of course).

That way, it someone relies on creating notifications when the bot fails, they can just look for a non-zero exit code (or parse the logs, I guess, but that would be more difficult).

I implemented these changes in 1.0.3rc8, feel free to give it try and check if it matches what you expected:
```
pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pleroma-bot==1.0.3rc8
```

**Comment by nemobis** [comment]
Il 03/04/22 20:23, robertoszek ha scritto:
> I think a good compromise it's to continue to execute and process the next users on the config but exit the app with an error exit code (and logging the exceptions and errors, of course).

Yes, that makes a lot of sense to me.
